vite.config.js

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import tsconfigPaths from "vite-tsconfig-paths";
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+export default defineConfig({
+  base: "",
+  root: "",
+  envPrefix: "RECORD_MANAGER_",
+  plugins: [react(), tsconfigPaths()],
+  server: {
+    port: 3000,
+  },
+  preview: {
+    port: 3000,
+  },
+  build: {
+    outDir: 'build',
```

Comment:
We want to go with a default setting of Vite as much as possible. 

On the other hand, we can modify Dockerfile to achieve the same.

Reply:
Ok, I set the outdir explicitly as 'dist', and changed the "build" to the "dist" in paths in the Dockerfile

Reply:
i would go with default value as in fta-fmea-ui -- https://github.com/kbss-cvut/fta-fmea-ui/blob/e1a243c59f12c55fae8fe66ad8b558bd104b0263/vite.config.js#L9

Reply:
I wanted to make it explicit, so that it envDir will not be accidentally set to another value later. But I can remove it, if it is not necessary

Reply:
believe you meant `outDir` instead of `envDir`. To me, if anybody changes the `outDir`,  we will immediately find out in GitHub action so that is why I do not care to have it explicit and instead keep the configuration of Vite simpler + same everywhere. Of course, this is almost a matter of taste.